### DISCUSSION TITLE
Indicazioni su annotazione modificativa su atto cartaceo

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#1225] [Supporto servizi cooperativi] [answered]
Buongiorno.
In quali circostanze è previsto l'utilizzo dei casi d'uso:

![image](https://github.com/user-attachments/assets/b05b9418-d51a-461b-af77-09368d524313)

introdotti con la versione 1.33?

**Comment by acolosimosogei** [comment]
Buongiorno, 

I casi d'uso proposti nella discussion consentono di registrare annotazioni destinate ad atti cartacei allo scopo di poterle utilizzare in formato digitale, semplificando lo scaricare e stampare e rendendo le annotazioni fisicamente attaccate all'atto, oppure per inviarle via pec se destinate ad atti cartacei di altri comuni. I casi d'uso sono stati inseriti per uniformare il comportamento già presente nel sistema in caso di annotazioni automaticamente generate destinate ad atti cartacei. E' stato concordato e deciso di registrare a sistema tutte le annotazioni non solo quelle digitali ma anche quelle destinate eventualmente ad atti pregressi.

Grazie e buona giornata,
Assistenza ANSC

**Comment by antonioluberto** [comment]
Cosa si intende con "registrare a sistema tutte le annotazioni non solo quelle digitali ma anche quelle destinate eventualmente ad atti pregressi"; se, come credo, l'atto pregresso è "cartaceo", come dovrebbe avvenire la registrazione - in ANSC - di annotazione apposta su un atto che in ANSC non c'è?
Possiamo avere la descrizione di un flusso di esempio in cui si possa/debba utilizzare uno dei casi d'uso in esame?

**Comment by acolosimosogei** [comment]
Il requisito concordato con ANUSCA e con il dipartimento consiste nel registrare a sistema anche annotazioni per atti cartacei al solo scopo di censirle nel sistema; queste annotazioni sono poi destinate alla stampa per poter essere attaccate fisicamente all'atto cartaceo. Non servono a molto altro se non a censire e archiviare le annotazioni nel sistema.

Grazie e buona serata,
Assistenza ANSC

**Comment by antonioluberto** [comment]
Possiamo avere la descrizione di un flusso di esempio in cui un Comune possa registrare a sistema anche annotazioni per atti cartacei ?

**Comment by asqueglia** [comment, marked as answer]
Come già specificato dalle risposte precedenti, i predetti casi di uso sono stati introdotti per compilare annotazioni da apporre su un atto esclusivamente cartaceo e non digitale allo scopo di poterle usare in formato digitale, per gestire quindi situazioni ibride in cui alcuni Comuni sono ANSC ed altri no.  Per avere una panoramica su flussi di esempio, consigliamo di vedere la guida operativa disponibile al link seguente 
https://www.anpr.interno.it/wp-content/uploads/Guida-operativa-ANSC.pdf (capitolo 16. GESTIONE OPERATIVA DEL COMUNE: ATTI DIGITALI E ATTI CARTACEI e successivi paragrafi)

Un esempio di utilizzo è un comune che subentra ad ANSC e ha bisogno di rettificare un atto di nascita cartaceo registrando una annotazione modificativa generica. E' stato richiesto come requisito, di consentire nel sistema ANSC la registrazione dell'annotazione destinata al cartaceo.

Chiudiamo pertanto la discussion
